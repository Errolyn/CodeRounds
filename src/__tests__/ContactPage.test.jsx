import ContactPage from '../contact/ContactPage';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders the contact page correctly', () => {
    const tree = renderer
      .create(<ContactPage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
});