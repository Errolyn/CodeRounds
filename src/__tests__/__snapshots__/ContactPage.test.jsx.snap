// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the contact page correctly 1`] = `
<div>
  Contact Page
</div>
`;
